functions.php

```diff
@@ -0,0 +1,36 @@
+<?php
+function pullAllAnimalsFromDb(): array
+{
+    $connectionString = 'mysql:host=db; dbname=endangeredlist';
+    $dbUsername = 'root';
+    $dbPassword = 'password';
+    $db = new PDO($connectionString, $dbUsername, $dbPassword);
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+    $queryString = 'SELECT * FROM `animals`;';
```

Comment:
Replace the * with actual names of columns (in backticks).

An * is not recommended in production code.